packages/runtime/src/helpers/edge.ts

```diff
@@ -57,12 +57,14 @@ export interface FunctionManifest {
         name?: string
         path: string
         cache?: 'manual'
+        generator?: string
```

Comment:
Isn't `generator` always defined?

Reply:
Yep generator will always be defined, thanks for spotting this.